drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -226,6 +510,8 @@ end
 
 local function device_init(driver, device)
   if device.network_type == device_lib.NETWORK_TYPE_MATTER then
+    device:set_component_to_endpoint_fn(component_to_endpoint)
```

Comment:
Do these functions still need to be moved here before the `initialize_switch` call, or do they work where they were before now that we have the deferred configure for the buttons in place?

I don't think it matters where these are called in the `init` function, I just want to make sure I understand why they needed to be moved and if they still need to be for some reason.